client/src/menu/spectator/SpectatorGameScreen.tsx

```diff
@@ -38,6 +41,7 @@ export default function SpectatorGameScreen(): ReactElement {
                     <HeaderMenu chatMenuNotification={false}/>
                 </div>
                 <GameScreenMenus />
+                {mobile && <MenuButtons chatMenuNotification={false}/>}
```

Comment:
mobile === true if your really based

sammy when he doesnt know what comments to leave